modules/gapi/perf/streaming/gapi_streaming_source_perf_tests.cpp

```diff
@@ -72,12 +83,17 @@ PERF_TEST_P_(VideoCapSourcePerfTest, TestPerformance)
 }
 
 INSTANTIATE_TEST_CASE_P(Streaming, OneVPLSourcePerfTest,
-                        Values(source_description_t(files[0], codec[0]),
-                               source_description_t(files[1], codec[1])));
+                        Values(source_description_t(files[0], codec[0], ""),
+                               source_description_t(files[0], codec[0], "MFX_ACCEL_MODE_VIA_D3D11"),
+                               source_description_t(files[1], codec[1], ""),
+                               source_description_t(files[1], codec[1], "MFX_ACCEL_MODE_VIA_D3D11"),
+                               source_description_t(files[2], codec[2], ""),
+                               source_description_t(files[2], codec[2], "MFX_ACCEL_MODE_VIA_D3D11")));
```

Comment:
for VPL sources in case of raw video stream input several files depends from codec, so must be pair file <->codec
But Accel Type may be combined with pair file <->codec. I'll try it later

Reply:
Sorry, I see. Maybe there are no benefits then. Will accept what you decide